backend/routes/misc.py

```
@@ -0,0 +1,37 @@
+    """Placeholder for fetching a user profile."""
+    # In production this would fetch from a database.
+    return {
+        "full_name": "Test User",
+        "birth_datetime": _dt.datetime(1990, 1, 1, 12, 0),
+        "latitude": 0.0,
+        "longitude": 0.0,
+    }
+
+
+@misc_bp.route("/report", methods=["GET"])
+def generate_report() -> "flask.Response":
```

Ruff (0.11.9)

Undefined name `flask`

(F821)